_specification-v1/sass.md

```diff
@@ -19,6 +14,10 @@ collection_listing_display: false
 
 Origami component styles are authored in <a href="http://sass-lang.com/" class="o-typography-link--external">Sass</a>, specifically the SCSS syntax. Sass features **should** be used only where they result in increased clarity and reuse. Care should be taken that the resulting CSS is not compromised by unnecessary Sass nesting.
 
+## Sass compiler
+
+Origami components **must** be able to be compiled with the <a href="https://sass-lang.com/dart-sass" class="o-typography-link--external">Dart-sass</a> compiler.
```

Comment:
```suggestion
Origami components **must** be able to be compiled with the primary implementation of Sass.
```

Reply:
👍 This is good. It gives us permission to remove `libsass` compilation tests from `origami-build-tools` for v2 components. We should add documentation to the migration guides which says to move to dart sass, the primary implementation

Reply:
I think we should put this back to "Dart Sass", and if a new implementation comes out that we want to use that we create a new specification that names it.

having it be "The Primary Implementation" means that components can become non-spec compliant just by sitting there.

this reminds me of how the javascript's specification of `.trim()` says that it must remove the unicode specification's definition of whitespace, which changes independently of the javascript specification. 

what do you think?

Reply:
That is an excellent point with a great real-world example 😅